lib/core.ps1

```diff
@@ -152,6 +152,11 @@ function file_path($app, $file) {
     return $null
 }
 
+function cmd_available($cmd) {
+    try { Get-Command $cmd -ea stop | out-null } catch { return $false }
+    $true
```

Comment:
Why not just one liner with bool cast??

```suggestion
    return [bool] (Get-Command $cmd -ErrorAction SilentlyContinue)
```

Or at least use return
```suggestion
    return $true
```
![image](https://user-images.githubusercontent.com/13260377/55792461-f3157080-5ac0-11e9-9493-4773ef29b946.png)

Reply:
Thanks, just move this function from `install.ps1`.